config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  resources :games
+  resources :games, only: [:index, :update, :destroy]
```

Comment:
Will this prevent your routes for the create API endpoint?

Reply:
We were testing each endpoint and :create is supposed to go after :index.